indico/modules/events/registration/controllers/management/regforms.py

```diff
@@ -214,24 +214,37 @@ class RHRegistrationFormOpen(RHManageRegFormBase):
     """Opens registration for a registration form"""
 
     def _process(self):
+        old_dts = (self.regform.start_dt, self.regform.end_dt)
         if self.regform.has_ended:
             self.regform.end_dt = None
         else:
             self.regform.start_dt = now_utc()
         logger.info("Registrations for %s opened by %s", self.regform, session.user)
         flash(_("Registrations for {} are now open").format(self.regform.title), 'success')
+        new_dts = (self.regform.start_dt, self.regform.end_dt)
+        if new_dts != old_dts:
+            if not old_dts[1]:
```

Comment:
This way I will not have track `old_dts` somewhere else in the code to see what the first element refers to
```suggestion
            if not self.regform.end_dt:
```